standalone/server.c

```diff
@@ -998,26 +998,14 @@ AP_DECLARE(void) unixd_pre_config(apr_pool_t *ptemp)
     apr_finfo_t wrapper;
 
 #if AP_SERVER_MAJORVERSION_NUMBER > 1 && AP_SERVER_MINORVERSION_NUMBER < 3
-#ifdef WAF_JSON_LOGGING_ENABLE
```

Comment:
I think this code is not related to the persistent lock itself. It makes a little bit confusing.

Reply:
This due to we need to set permission for the lock, since we have now own lock utility, so we don't need to have the old set permission code, as you say, i will split it into two pr, so it will be included in the integration of lock